proto/query.proto

```diff
@@ -476,7 +477,9 @@ message CommitRequest {
 }
 
 // CommitResponse is the returned value from Commit
-message CommitResponse {}
+message CommitResponse {
+  int64 reserved_id = 1;
```

Comment:
Why do we need to return this for Commit and Rollback?

Reply:
Because a commit will cause the id to change, to prevent reuse of ids.